dracut/30ignition/coreos-remove-initramfs-network.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+# -*- mode: shell-script; indent-tabs-mode: nil; sh-basic-offset: 4; -*-
+# ex: ts=8 sw=4 sts=4 et filetype=sh
+
+# Clean up the interfaces set up in the initramfs
+for f in /sys/class/net/*; do
+    interface=$(basename "$f")
+    ip link set $interface down
+    ip addr flush dev $interface
+    rm -f -- /tmp/net.$interface.did-setup
```

Comment:
Is this needed?

Reply:
Hmm the file does exist in the initramfs, but it does not seem to cause errors if not removed. I just followed dracut's `ifdown`

Reply:
Hmm OK, I see it now. Let's maybe add a comment at the top of the script that this is mimicking dracut's `ifdown()`?